### PR TITLE
Adding Dapr Grafana dashboards as part of release

### LINKED ISSUE #1840: Add Dapr Grafana dashboards as part of release
Dapr Grafana dashboards are placed in https://github.com/dapr/docs/tree/master/reference/dashboard folder.

This issue is to make them versioned and part of official releases.

### PR DESCRIPTION
**Pull request by shalabhms**
Dapr Grafana dashboards are placed in https://github.com/dapr/docs/tree/master/reference/dashboard folder.

This PR is to make them part of official releases.

## Issue reference

Closes #1840 

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [ ] Created/updated tests
* [x] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by shalabhms** [comment]
image

File: .github/workflows/dapr.yml [hunk 1]
```diff
@@ -71,6 +71,11 @@ jobs:
         with:
           name: dapr_${{ matrix.target_os }}_${{ matrix.target_arch }}
           path: ${{ env.ARCHIVE_OUTDIR }}
+      - name: upload grafana dashboards artifacts
+        uses: actions/upload-artifact@master
+        with:
+          name: dapr_grafana_dashboards
+          path: ./grafana/
```
Comment (youngbupark) [inline .github/workflows/dapr.yml:78]: I thought we would want to upload it to github release. this uploads files to github actions build artifact.

Comment (shalabhms) [reply .github/workflows/dapr.yml:78]: @youngbupark , When the release is cut,  wont this upload to github release ?

Comment (shalabhms) [reply .github/workflows/dapr.yml]: Created the new workflow which will upload grafana dashboards to github release.

File: .github/workflows/dapr.yml [hunk 2]
```diff
@@ -71,6 +71,12 @@ jobs:
         with:
           name: dapr_${{ matrix.target_os }}_${{ matrix.target_arch }}
           path: ${{ env.ARCHIVE_OUTDIR }}
+      - name: upload artifacts - grafana dashboards
+        if: matrix.target_arch == 'amd64' && matrix.target_os == 'linux'
+        uses: actions/upload-artifact@master
+        with:
+          name: dapr_grafana_dashboards
+          path: ./grafana/
```
Comment (youngbupark) [inline .github/workflows/dapr.yml:79]: it will upload readme.md in grafana directory. use `./grafana/*.json`

**Pull request merged**